configure-windows

```diff
@@ -7,16 +7,8 @@
 
 
 export prefix=C:/ocamlmgw64
-while : ; do
-    case "$1" in
-        "") break;;
-        -prefix|--prefix)
-            prefix=$2; shift;;
-    esac
-    shift
-done
 
 echo "[configure-windows] Setting up flexdll"
-./clone-flexdll
-./configure --build=x86_64-unknown-cygwin --host=x86_64-w64-mingw32 --prefix="$prefix"
-make flexdll
+. ./clone-flexdll
+./configure --build=x86_64-unknown-cygwin --host=x86_64-w64-mingw32 --prefix="$prefix" "$@"
+exec make flexdll
```

Comment:
Unrelated remark: shouldn't this call to make be in esy-build ?

Reply:
I believe it should also had this problem in the past.

Reply:
https://github.com/ocaml/ocaml/commit/731440fe59bdaa118a1f06f99e8c86e56396ce3b definitive answer: we should remove it !!!!